kernel/cycle.h

```diff
@@ -1,6 +1,7 @@
 /*
  * Copyright (c) 2003, 2007-14 Matteo Frigo
  * Copyright (c) 2003, 2007-14 Massachusetts Institute of Technology
+ * Copyright (c) 2024 Christopher Taylor, Tactical Computing Labs, LLC
```

Comment:
MIT will require a copyright assignment to merge PRs like this.

Reply:
not an issue, can you send or post information on how that process works?